docs/application/native/guides/app-management/app-controls.md

```diff
@@ -664,6 +668,123 @@ create_base_gui(appdata_s *ad)
    }
    ```
 
+<a name="action"></a>
+## Handling application control actions
```

Comment:
Please keep heading in title case. Kindly rephrase this line as "Handle Application Control Actions"

Reply:
Done

Reply:
> @hjhun Please keep heading in title case. Kindly write it as "Handle Application Control Actions"

@hjhun Kindly incorporate this comment also.

Reply:
Done